src/components/Banner/Banner.styles.ts

```diff
@@ -1,35 +1,35 @@
 import styled from "styled-components";
 
-export const IntroContainerStyled = styled.main`
+export const BannerContainerStyled = styled.main`
```

Comment:
We can leave it like this without any problem, but the standard naming for styled components would be **StyledBannerContainer** because in english makes sense the adjective coming first.

Reply:
I'm creating another card to review the whole project